src/kern/arm/bsp/s32z/mru-arm-s32z.cpp

```diff
@@ -0,0 +1,94 @@
+INTERFACE [arm && pic_gic && pf_s32z]:
+
+#include "gic.h"
+#include "gic_v3.h"
+#include "mmio_register_block.h"
```

Comment:
These headers are already included in the generic part. No need to repeat them here.